### ISSUE TITLE
Magisk19.1 makes MIUI10 run slower than before.

### ISSUE THREAD

**Issue by dkizjun** [topjohnwu/Magisk#1410]
It's just like the title.But  Magisk19.0 not have this problem.Please fix it.

**Comment by raphCode** [comment]
I also found that Magisk 19.1 slows down my device. I have an old device (Samsung Galaxy Tab 2) so the performance hit makes using the system almost unuseable

I currently use LineageOS 13, so its Android 6.
Oberservation through htop showed that there were two system_server processes that use 100% CPU each, utilizing the dual core processor to the max. BetterBatteryStats showed nothing conspicuous. Deep Sleep was indeed entered on screen off, but the constant CPU of course causes bad battery life.

Downgrading to Magisk 19.0 or disabling it (restoring stock boot image) solves the problem

**Comment by raphCode** [comment]
Confirmed behaviour on Nougat / linageOS 14.1, same device.
System also runs very slow with Magisk 19.1, but so far fine with 19.0. With the problematic version, I was not able to see any process with high CPU load.

I did not know previously that the lag relates to Magisk, since I also did not find any process that consumed much CPU and the nougat Lineage port for this device is said to be prone to lagging anyway.
But with a downgraded Magisk (19.0) it runs super smooth!
